packages/catalyst_dapp_connector/lib/src/dapp_connector.dart

```diff
@@ -0,0 +1,63 @@
+@JS('cardano')
+import 'package:js/js.dart';
+import 'package:js/js_util.dart';
+import 'package:catalyst_dapp_connector/src/models/wallet_model.dart';
+import 'package:catalyst_dapp_connector/src/models/web_wallet_model.dart';
+import 'package:catalyst_dapp_connector/src/models/voting_credentials_model.dart';
+import 'package:catalyst_dapp_connector/src/models/cip62_api_model.dart';
+
+external WebWallet get flint;
+external WebWallet get nami;
+external WebWallet get ccvault;
+external WebWallet get gerowallet;
+external WebWallet get nufi;
+
+// TODO: Is there a way to not hardcode wallets?
+WebWallet getWalletApi(String name) {
+  switch (name) {
+    case 'nufi':
+      return nufi;
+    case 'flint':
+      return flint;
+    default:
+      return nufi;
+  }
+}
+
+Wallet version(String walletName) {
+  try {
+    return Wallet.formWebWallet(getWalletApi(walletName));
+  } catch (e) {
+    throw Exception('Error getting wallet version: $e');
+  }
+}
+
+// TODO: refactor from line 15
```

Comment:
I'm confused.  The TODO on line 35 says we need to refactor from line 15. What exactly do we need to refactor, and why not implement it correctly?

Reply:
@minikin The comment is referring to the fact that we had to hard-code, put in a list, every wallet we want to use. We were thinking about finding more dynamical way, getting the wallet name from the browser's "window". And again, this early in the dev, we use comments as reminders, to highlight parts of the code